SLA.R

```diff
@@ -6,5 +6,5 @@ print(sla)
 print(summary(sla))
 
 library(ggplot2)
-p <- qplot(n_Leaves, Leaf.Area..cm2., data = sla)
+p <- qplot(n_Leaves, Leaf_Area_.cm2., data = sla)
```

Comment:
Hmm, I don't think will work--did you try running it? The column name is `Leaf_Area_cm2` (no periods)